test/db/analysis/vars

```diff
@@ -1141,28 +1141,7 @@ EXPECT=<<EOF
 |       |   0x8000050c      ld.a  a15, [a15]#0
 |       |   0x8000050e      ld.w  d15, [a14]#-4
 |       |   0x80000512      st.w  [a15]#0, d15
-|       |   0x80000514      ld.a  a15, [a14]#-0x18
-|       |   0x80000518      nop
-|       |   0x8000051a      ld.w  d15, [a15]#0
-|       |   0x8000051c      mov.a a15, d15
-|       |   0x8000051e      add.a a15, #4
-|       |   0x80000520      mov   d4, #5
-|       |   0x80000522      ld.a  a5, [a14]#-8
-|       |   0x80000526      mov.aa a4, a15
-|       |   0x80000528      call  dbg.strncpy
-|       |   0x8000052c      ld.a  a15, [a14]#-0x18
-|       |   0x80000530      ld.a  a15, [a15]#0
-|       |   0x80000532      ld.d  e4, [a14]#-0x10
-|       |   0x80000536      call  dbg.__truncdfsf2
-|       |   0x8000053a      mov   d15, d2
-|       |   0x8000053c      st.w  [a15]#0xc, d15
-|       |   0x8000053e      ld.a  a15, [a14]#-0x18
-|       |   0x80000542      ld.a  a15, [a15]#0
-|       |   0x80000544      ld.w  d15, [a14]#-0x14
-|       |   0x80000548      st.w  [a15]#0x10, d15
-|       |   0x8000054a      mov   d15, #1
-|       `-> 0x8000054c      mov   d2, d15
-\           0x8000054e      ret
+\       |   0x80000514      ld.a  a15, [a14]#-0x18
```

Comment:
This looks like a regression. before it was reaching the `ret`, but now it stops at `ld.a`

Reply:
Yes, it does. But The call addresses seem correct. I don't know enough about TriCore. Maybe they compile overlapping functions.

Reply:
See my reply in mattermost.